### ISSUE TITLE
Dark theme for dashboard

### ISSUE THREAD

**Issue by hakong** [opinkerfi/adagios#665]
Something similar to Grafana's dark theme color palette?
![image](https://user-images.githubusercontent.com/4422113/85536520-b15be180-b602-11ea-97c0-5250715de4b7.png)

![image](https://user-images.githubusercontent.com/4422113/85535763-03e8ce00-b602-11ea-9aed-b09181a9902b.png)

**Comment by gardart** [comment]
I agree , Adagios should have more themes like that. I will try to add this for the next build (after porting to Py3)

Kv Garðar

**Comment by hakong** [comment]
Superb. Thanks!
We are trying to make all of our wall monitors dark-themed and Adagios was the only one that didn't work well with [Dark Reader](https://chrome.google.com/webstore/detail/dark-reader/eimadpbcbfnmbkopoojfekhnkhdbieeh?hl=en)
